### ISSUE TITLE
Colab keeps crashing,

### ISSUE THREAD

**Issue by FarazNadeem22** [googlecolab/colabtools#1441]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior: After running a few lines of code my colab session keeps crashing.  

- Describe the expected behavior: Do not crash if I defined X in a cell above I should be able to call it 2 mins later in the cell bellow. too much to ask?

- The web browser you are using (Chrome, Firefox, Safari, etc.): Chrome 

- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):

**Comment by craigcitro** [comment]
Please include a minimal, public, self-contained repro notebook.

**Comment by FarazNadeem22** [comment]
!pip install tensorflow-gpu==2.2.0
  !pip install keras-lookahead
  !pip install keras-radam

import os
path='/content/drive/My Drive/HES/HW5/HW4_Data'
os.path=path
os.chdir(path=path)
os.listdir()

import gc, sys, os
import numpy as np
from matplotlib import pyplot as plt
from sklearn import model_selection
import keras
from keras.utils import to_categorical
from keras.layers import Input, Conv2D, MaxPooling2D, Dense, Flatten, Dropout, BatchNormalization, Activation, GlobalAveragePooling2D
from keras.optimizers import SGD, Adam, RMSprop
from keras.callbacks import EarlyStopping, ModelCheckpoint
from keras.models import load_model, Model
from keras.regularizers import l2
from keras.preprocessing.image import ImageDataGenerator
from keras_radam import RAdam
from keras_lookahead import Lookahead
from keras import layers  
from keras import losses 

def load_dataset():
# load dataset
    npx = np.load('dc_10k_photos_128.npy')
    npy = np.load('dc_10k_labels_128.npy')
    X, X_test, y, y_test = model_selection.train_test_split(npx, npy, test_size=0.9, random_state=13)
    # garbage collection or run out of RAM
    npx = None
    npy = None
    gc.collect()
    X_train, X_val, y_train, y_val = model_selection.train_test_split(X, y, test_size=0.5, random_state=17)
    # one hot encode target values
    y_train = to_categorical(y_train)
    y_val = to_categorical(y_val)
    y_test = to_categorical(y_test)
    print(X_train.shape, X_val.shape, X_test.shape)
    print(y_train.shape, y_val.shape, y_test.shape)
    # convert from integers to floats
    X_train = X_train / 255.0
    X_val =  X_val / 255.0
    X_test = X_test / 255.0
    return X_train, y_train, X_val, y_val, X_test, y_test

X_train, y_train, X_val, y_val, X_test, y_test = load_dataset()

from keras.applications.resnet50 import ResNet50
resnet = keras.applications.ResNet101V2(
    include_top=True,
    weights="imagenet",
    # input_shape = (128,128,3)
)

**Comment by craigcitro** [comment]
I suspect you're hitting Drive API quota limits; see #1020 for some potential workarounds.

**Comment by FarazNadeem22** [comment]
thanks. I am in colab pro.
